src/tools/copy/copytool.cpp

```diff
@@ -16,7 +16,7 @@ bool CopyTool::closeOnButtonPressed() const
 
 QIcon CopyTool::icon(const QColor& background, bool inEditor) const
 {
-    Q_UNUSED(inEditor);
+    Q_UNUSED(inEditor)
```

Comment:
I assume you removed all the macro semicolons because an IDE suggested it? Just curious which IDE suggested it and why?

Reply:
I did it by recommendation of IDE CLion, decrease number of warnings, it is redundant.